Update threat-analytics.md (#7444)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/threat-analytics.md

![Image of a threat analytics dashboard.](images/ta_dashboard.png)

## View the threat analytics dashboard

> The threat analytics dashboard ([security.microsoft.com/threatanalytics3](https://security.microsoft.com/threatanalytics3)) highlights the reports that are most relevant to your organization. It summarizes the threats in the following sections:

Reviewer: ```suggestion
The [threat analytics dashboard](https://security.microsoft.com/threatanalytics3) highlights the reports that are most relevant to your organization. It summarizes the threats in the following sections:
```

- **Latest threats**—lists the most recently published or updated threat reports, along with the number of active and resolved alerts.

> - **High-impact threats**—lists the threats that have the highest impact to your organization. This section lists threats with the highest number of active and resolved alerts first.

Reviewer: ```suggestion
- **High-impact threats**—lists the threats that have the highest impact on your organization. This section lists threats with the highest number of active and resolved alerts first.
```

- **Threat summary**—provides the overall impact of all tracked threats by showing the number of threats with active and resolved alerts.

Select a threat from the dashboard to view the report for that threat.

![Screenshot of threat analytics dashboard.](../../media/threat-analytics/ta_dashboard_mtp.png)

> _Threat analytics dashboard. You can also click the Search icon to key in a keyword related to the threat analytics report that you'd like to read._ 

Reviewer: ```suggestion
_Threat analytics dashboard. You can also select the **Search** icon to key in a keyword related to the threat analytics report that you would like to read._ 
```

- [**Analyst report**](#analyst-report-get-expert-insight-from-microsoft-security-researchers)
- [**Related incidents**](#related-incidents-view-and-manage-related-incidents)
- [**Impacted assets**](#impacted-assets-get-list-of-impacted-devices-and-mailboxes)
- [**Prevented email attempts**](#prevented-email-attempts-view-blocked-or-junked-threat-emails)
- [**Mitigations**](#mitigations-review-list-of-mitigations-and-the-status-of-your-devices)

### Overview: Quickly understand the threat, assess its impact, and review defenses

The **Overview** section provides a preview of the detailed analyst report. It also provides charts that highlight the impact of the threat to your organization and your exposure through misconfigured and unpatched devices.

![Image of the overview section of a threat analytics report.](../../media/threat-analytics/ta_overview_mtp.png)

_Overview section of a threat analytics report_

#### Assess impact on your organization

Each report includes charts designed to provide information about the organizational impact of a threat:

- **Related incidents**—provides an overview of the impact of the tracked threat to your organization with the following data:

>   - Number of active alerts and the number of active incidents they are associated with
>   - Severity of active incidents

Reviewer: ```suggestion
  - Number of active alerts and the number of active incidents they are associated with.
  - Severity of active incidents.
```

_Impacted assets section of a threat analytics report_

### Prevented email attempts: View blocked or junked threat emails

Microsoft Defender for Office 365 typically blocks emails with known threat indicators, including malicious links or attachments. In some cases, proactive filtering mechanisms that check for suspicious content will instead send threat emails to the junk mail folder. In either case, the chances of the threat launching malware code on the device is reduced.

The **Prevented email attempts** tab lists all the emails that have either been blocked before delivery or sent to the junk mail folder by Microsoft Defender for Office 365. 

![Image of the prevented email attempts section of a threat analytics report.](../../media/threat-analytics/ta_prevented_email_attempts_mtp.png)

_Prevented email attempts section of a threat analytics report_

### Mitigations: Review list of mitigations and the status of your devices

In the **Mitigations** section, review the list of specific actionable recommendations that can help you increase your organizational resilience against the threat. The list of tracked mitigations includes:

> - **Security updates**—deployment of supported software security updates for vulnerabilities found on onboarded devices

Reviewer: ```suggestion
- **Security updates**—deployment of supported software security updates for vulnerabilities found on onboarded devices.
```
